src/Arcus.WebApi.Security/Authorization/Jwt/JwtTokenReader.cs

```diff
@@ -53,8 +70,8 @@ public JwtTokenReader(TokenValidationParameters tokenValidationParameters, strin
             Guard.NotNullOrWhitespace(openIdConnectDiscoveryUri, nameof(openIdConnectDiscoveryUri));
             Guard.NotNull(tokenValidationParameters, nameof(tokenValidationParameters));
 
-            _tokenValidationParameters = tokenValidationParameters;
-            _configManager = new ConfigurationManager<OpenIdConnectConfiguration>(openIdConnectDiscoveryUri, new OpenIdConnectConfigurationRetriever());
+            this._tokenValidationParameters = tokenValidationParameters;
```

Comment:
You can leave this out
```suggestion
            _tokenValidationParameters = tokenValidationParameters;
```

Reply:
Done